tools/rosgraph/src/rosgraph/xmlrpc.py

```diff
@@ -345,6 +357,7 @@ def _run_init(self):
             else:
                 msg = "ERROR: Unable to start XML-RPC server: %s" % e.strerror
             logger.error(msg)
+            print(msg)
```

Comment:
This was already there before Secure ROS (I had removed it - but then re-added it to remove noise)